plasma_framework/test/src/framework/ExitGameController.test.js

```diff
@@ -458,4 +479,40 @@ contract('ExitGameController', () => {
             );
         });
     });
+
+    describe('find exit in the priority queue', () => {
+        beforeEach(async () => {
+            this.controller.addToken(this.dummyToken);
+        });
+
+        it('should be able to find the exit in the priority queue', async () => {
+            const dummyExit = {
+                token: this.dummyToken,
+                exitableAt: 1,
+                txPos: buildTxPos(1000, 1),
+                exitId: 456,
+                exitProcessor: this.dummyExitGame.address,
+            };
+            const { tx } = await this.dummyExitGame.enqueue(
+                dummyExit.token,
+                dummyExit.exitableAt,
+                dummyExit.txPos,
+                dummyExit.exitId,
+                dummyExit.exitProcessor,
+            );
+
+            // Get the exit's uniquePriority from the ExitQueued event
```

Comment:
If we want to mimic what a client would be doing, I guess we have to search with `exitId` here, this assumes given tx, we can know exitId accordingly.

Or I guess we can just get receipt and log from the previous line if we are not aiming to mimic client behavior?
```
const {receipt, log} = await this.dummyExitGame.enqueue(...);
```

I am not really sure what's your intention while designing the test so I will leave this to your decision.

Reply:
Yes, the intention was to mimic client behaviour - I was just taking a shortcut to get the logs.
Your way is closer to what a real client would do though...

Reply:
> Otherwise I guess we can just get receipt and log from the previous line if we are not aiming to mimic client behavior?
`const {receipt, log} = await this.dummyExitGame.enqueue(...);`

BTW This doesn't work because the `log` returned here is empty.

I _think_ that this is because it only contains any logs written by the 'called' transaction (i.e. `dummyExitGame.enqueue()`) and not by any internal transactions (i.e.`ExitGameController.enqueue()`).

Not sure if this is a bug or intentional behaviour...